dev-python/plotly/plotly-4.10.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# blocker dev-python/retrying
```

Comment:
Got a bug for it? ;)

Reply:
`retrying ` is `python3_{6,7}` only hence I can't add `8,9` as of now.